### ISSUE TITLE
The applicationDictionary dialog fails to display on 2026.05.09, and also causes NVDA to not fully speak the control information when system focus automatically lands on certain dialogs and windows

### ISSUE THREAD

**Issue by Joker-Woo** [ruifontes/applicationDictionary-#11]
#### First abnormal phenomenon

With applicationDictionary 2026.05.09 enabled, press NVDA+N to open the NVDA menu, select Preferences subMenu, then select Speech dictionaries subMenu, then select Application Dictionary. Normally the applicationDictionary dialog would appear. But now, after pressing Enter on the Application Dictionary menu item, the NVDA menu disappears as expected, yet strangely no dialog pops up.

#### Second abnormal phenomenon

Enabling applicationDictionary 2026.05.09 seems to cause NVDA to not fully speak the control information when system focus automatically lands on certain dialogs and windows.

For example, after pressing NVDA+Q to show the Exit NVDA dialog, normally NVDA would speak the following information:

```
Exit NVDA  dialog
What would you like to do?  combo box  Exit  collapsed  Alt+  d
```

But with applicationDictionary 2026.05.09 enabled, pressing NVDA+Q to show the Exit NVDA dialog causes NVDA to speak only:

```
Exit NVDA  dialog
collapsed
```

Also, when closing or minimizing certain windows, such as Firefox, if the system focus then lands on the desktop or taskbar, NVDA would normally speak the control information that received focus. With applicationDictionary 2026.05.09 enabled, NVDA speaks nothing.

***

* NVDA version: 2026.1rc1 (2026.1.0.55678)	
* applicationDictionary version: 2026.05.09

I have tested this on a completely clean NVDA portable with no add-ons and no personal settings, and the issues above are still reproducible.

#### Below is the NVDA log snippet

```
IO - speech.speech.speak (20:31:45.769) - MainThread (11408):
Speaking [LangChangeCommand ('zh_CN'), 'Log fragment start position marked, press again to copy to clipboard']
IO - inputCore.InputManager.executeGesture (20:31:48.733) - winInputHook (3108):
Input: kb(laptop):NVDA+n
DEBUG - gui.SysTrayIcon.evaluateUpdatePendingUpdateMenuItemCommand (20:31:48.733) - MainThread (11408):
Error while removing pending update menu item
Traceback (most recent call last):
  File "gui\__init__.pyc", line 743, in evaluateUpdatePendingUpdateMenuItemCommand
wx._core.wxAssertionError: C++ assertion ""node"" failed at ..\..\src\common\menucmn.cpp(462) in wxMenuBase::Remove(): removing item not in the menu?
ERROR - eventHandler.executeEvent (20:31:48.753) - MainThread (11408):
error executing event: gainFocus on <NVDAObjects.IAccessible.ContentGenericClient object at 0x000001F4014B8B50> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 354, in executeEvent
  File "eventHandler.pyc", line 120, in __init__
  File "eventHandler.pyc", line 129, in next
  File "C:\Users\qisha\NVDA-portable\userConfig\addons\applicationDictionary\globalPlugins\applicationDictionary\__init__.py", line 127, in event_gainFocus
    dict = getDict(appName)
  File "C:\Users\qisha\NVDA-portable\userConfig\addons\applicationDictionary\globalPlugins\applicationDictionary\__init__.py", line 86, in getDict
    return loadDict(appName)
  File "C:\Users\qisha\NVDA-portable\userConfig\addons\applicationDictionary\globalPlugins\applicationDictionary\__init__.py", line 73, in loadDict
    dict = speechDictHandler.types.SpeechDict()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "speechDictHandler\__init__.pyc", line 28, in __getattr__
AttributeError: module 'speechDictHandler' has no attribute 'types'
IO - speech.speech.speak (20:31:48.757) - MainThread (11408):
Speaking [LangChangeCommand ('zh_CN'), 'NVDA', 'menu', CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (20:31:50.401) - winInputHook (3108):
Input: kb(laptop):downArrow
IO - speech.speech.speak (20:31:50.408) - MainThread (11408):
Speaking [LangChangeCommand ('zh_CN'), 'Preferences', 'subMenu', CharacterModeCommand(True), 'p', CharacterModeCommand(False), CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (20:31:51.493) - winInputHook (3108):
Input: kb(laptop):rightArrow
IO - speech.speech.speak (20:31:51.508) - MainThread (11408):
Speaking [LangChangeCommand ('zh_CN'), 'Settings...', CharacterModeCommand(True), 's', CharacterModeCommand(False), CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (20:31:52.873) - winInputHook (3108):
Input: kb(laptop):downArrow
IO - speech.speech.speak (20:31:52.882) - MainThread (11408):
Speaking [LangChangeCommand ('zh_CN'), 'Speech dictionaries', 'subMenu', CharacterModeCommand(True), 'd', CharacterModeCommand(False), CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (20:31:54.707) - winInputHook (3108):
Input: kb(laptop):rightArrow
IO - speech.speech.speak (20:31:54.723) - MainThread (11408):
Speaking [LangChangeCommand ('zh_CN'), 'Default dictionary...', CharacterModeCommand(True), 'd', CharacterModeCommand(False), CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (20:31:55.671) - winInputHook (3108):
Input: kb(laptop):upArrow
IO - speech.speech.speak (20:31:55.678) - MainThread (11408):
Speaking [LangChangeCommand ('zh_CN'), 'Application Dictionary...', CharacterModeCommand(True), 'a', CharacterModeCommand(False), CancellableSpeech (still valid)]
IO - inputCore.InputManager.executeGesture (20:31:59.929) - winInputHook (3108):
Input: kb(laptop):enter
ERROR - unhandled exception (20:31:59.935) - MainThread (11408):
Traceback (most recent call last):
  File "C:\Users\qisha\NVDA-portable\userConfig\addons\applicationDictionary\globalPlugins\applicationDictionary\__init__.py", line 140, in script_editDict
    apDict = getDict(appName)
  File "C:\Users\qisha\NVDA-portable\userConfig\addons\applicationDictionary\globalPlugins\applicationDictionary\__init__.py", line 86, in getDict
    return loadDict(appName)
  File "C:\Users\qisha\NVDA-portable\userConfig\addons\applicationDictionary\globalPlugins\applicationDictionary\__init__.py", line 73, in loadDict
    dict = speechDictHandler.types.SpeechDict()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "speechDictHandler\__init__.pyc", line 28, in __getattr__
AttributeError: module 'speechDictHandler' has no attribute 'types'
DEBUGWARNING - IAccessibleHandler.internalWinEventHandler._shouldGetEvents (20:31:59.971) - MainThread (11408):
Foreground took too long to change. Foreground still 131088 (Progman). Should be 2033988 (wxWindowNR)
ERROR - eventHandler.executeEvent (20:31:59.984) - MainThread (11408):
error executing event: gainFocus on <NVDAObjects.IAccessible.sysListView32.List object at 0x000001F4014B8F50> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 354, in executeEvent
  File "eventHandler.pyc", line 120, in __init__
  File "eventHandler.pyc", line 129, in next
  File "C:\Users\qisha\NVDA-portable\userConfig\addons\applicationDictionary\globalPlugins\applicationDictionary\__init__.py", line 127, in event_gainFocus
    dict = getDict(appName)
  File "C:\Users\qisha\NVDA-portable\userConfig\addons\applicationDictionary\globalPlugins\applicationDictionary\__init__.py", line 86, in getDict
    return loadDict(appName)
  File "C:\Users\qisha\NVDA-portable\userConfig\addons\applicationDictionary\globalPlugins\applicationDictionary\__init__.py", line 73, in loadDict
    dict = speechDictHandler.types.SpeechDict()
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "speechDictHandler\__init__.pyc", line 28, in __getattr__
AttributeError: module 'speechDictHandler' has no attribute 'types'
DEBUG - utils.schedule.callJobOnThread (20:32:00.110) - ScheduleThread (2444):
Starting thread for job: _checkForUpdatableAddons on thread None
DEBUG - gui.addonStoreGui.controls.messageDialogs.UpdatableAddonsDialog._checkForUpdatableAddons (20:32:00.111) - _checkForUpdatableAddons (9964):
checking for updatable add-ons
DEBUG - addonStore.dataManager._DataManager._getCacheHash (20:32:00.111) - _checkForUpdatableAddons (9964):
Fetching add-on data from https://addonStore.nvaccess.org/cacheHash.json
DEBUG - gui.addonStoreGui.controls.messageDialogs.UpdatableAddonsDialog._checkForUpdatableAddons (20:32:01.037) - _checkForUpdatableAddons (9964):
no updatable add-ons found
IO - inputCore.InputManager.executeGesture (20:32:11.864) - winInputHook (3108):
Input: kb(laptop):NVDA+control+shift+f1
```
